game/graphics/mesh/animation.cpp

```diff
@@ -263,7 +263,7 @@ bool Animation::Sequence::isInComboWindow(uint64_t t, uint16_t comboLen) const {
 
 bool Animation::Sequence::isDefParWindow(uint64_t t) const {
   if(data->defParFrame.size()!=2)
-    return false;
+    return true;
```

Comment:
`Pose::isDefence` uses `isDefWindow`, not `isDefParWindow`.

Also, after checking the source, I've find out that `isDefParWindow` is not in use. Was this change made by mistake or this is some kind of leftover?

Reply:
`isDefWindow` is also used for attack anis which made no sense for me in parade if there's `DEF_PAR_FRAME` extra for it. Though this is never used for humans so my plan was to just check the animation and leave def window/parade out completely. Decided to check against `isDefParWindow` later in case this is actually used somewhere. At least form testing it felt parade was always blocking.

Did some search now for `DEF_WINDOW` and modding wiki gives:
> Set a “window” in the animation - an interval of frames during which you need to press the “up” (G1) or the left mouse button (G2) to continue the combo strike.

https://gothic-modding-community.github.io/gmc/zengin/anims/events/#def_window